### PR TITLE
Adding a metric and a metric bound to the churn prediction workflow

### PR DESCRIPTION
**Pull request by jegonzal**
This PR changes the check computation to validate probabilities between 0 and 1 and instead uses a metric to compute the average churn and set multiple bounds on the value of that churn.

### CODE REVIEW

**Comment by jegonzal** [comment]
We should probably rebuild the gitbook docs with the new version of this notebook.

File: sdk/examples/churn_prediction/Build and Deploy Churn Ensemble.ipynb [hunk 1]
```diff
@@ -712,7 +712,7 @@
    "outputs": [],
```
Comment (vsreekanti) [inline sdk/examples/churn_prediction/Build and Deploy Churn Ensemble.ipynb:712]: <p>"An Aqueduct <code>check</code>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aqueducthq/aqueduct/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

**Comment by vsreekanti** [comment]
This looks good to me -- there's just a small typo in some of the text.

File: sdk/examples/churn_prediction/Build and Deploy Churn Ensemble.ipynb (hunk 1 above)
Comment (jegonzal) [reply sdk/examples/churn_prediction/Build and Deploy Churn Ensemble.ipynb:712]: Resolved. Can you accept?

**Review by vsreekanti — APPROVED** [review]

**Pull request merged**